Services/WebServices/ECS/classes/class.ilAuthProviderECS.php

```diff
@@ -201,33 +215,7 @@ public function validateHash()
 			
 			$this->getLogger()->dump($auths, ilLogLevel::DEBUG);
 
-			if($auths->pid)
-			{
-				try
-				{
-					include_once './Services/WebServices/ECS/classes/class.ilECSCommunityReader.php';
-					$reader = ilECSCommunityReader::getInstanceByServerId($this->getCurrentServer()->getServerId());
-					$part = $reader->getParticipantByMID($auths->pid);
```

Comment:
Yes, this was wrong. 
Using the $auths->pid as mid will procude wrong results if participants are assigned to more than one community.

Reply:
Instead I will add the new method $reader->getParticipantsByPid($pid) which will be used to receive the current abreviation.